frontend/src/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createBrowserHistory } from 'history';
+import "./index.css"
+
+import ExperimentWindow from './MainWindow.js';
```

Comment:
```suggestion
import MainWindow from './MainWindow.js';
```

or

```suggestion
import ExperimentWindow from './ExperimentWindow.js';
```

Name should be consistent